server/src/test/java/org/elasticsearch/search/fetch/subphase/FetchSourcePhaseTests.java

```diff
@@ -190,6 +191,8 @@ private HitContext hitExecuteMultiple(
         SearchExecutionContext sec = mock(SearchExecutionContext.class);
         when(sec.isSourceEnabled()).thenReturn(sourceBuilder != null);
         when(fetchContext.getSearchExecutionContext()).thenReturn(sec);
+        MappingLookup mappingLookup = MappingLookup.EMPTY;
+        when(fetchContext.getSearchExecutionContext().getMappingLookup()).thenReturn(mappingLookup);
```

Comment:
Nit: no need for the `mappingLookup` var

```suggestion
        when(fetchContext.getSearchExecutionContext().getMappingLookup()).thenReturn(MappingLookup.EMPTY);
```